.vsconfig

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.6.2.TargetingPack",
```

Comment:
This file being added is potentially problematic. I'm not sure why it added it, but if possible can you remove it from the commit and then add `.vsconfig` to the gitignore. 

We dont all run the same targeting pack so this might cause some people to have errors when trying to build